spec/javascripts/track-briefings.spec.js

```diff
@@ -0,0 +1,31 @@
+/* global describe beforeEach it spyOn expect */
+
+var $ = window.jQuery
+
+describe('Track Briefing', function () {
+
+  var GOVUK = window.GOVUK
+  var tracker
+  var element
+
+  GOVUK.analytics = GOVUK.analytics || {}
+  GOVUK.analytics.trackEvent = function () {}
+
+  beforeEach(function () {
+    spyOn(GOVUK.analytics, 'trackEvent')
+
+    element = document.createElement('div')
+    element.innerHTML = '<a href="/blah/blahhhh">Blahh</a>'
+
+    tracker = new GOVUK.Modules.TrackBriefing()
+    tracker.start($(element))
+  })
+
+  it('send ga event when link is clicked', function () {
+    element.querySelector('a').dispatchEvent(new Event('click'))
+
+    expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith(
+      'Briefings page', 'Blahh', { transport: 'beacon', label: "/blah/blahhhh" }
+    )
+  })
```

Comment:
Probably be good to have a test with multiple links to make sure its sending multiple events.

Reply:
Done